dls-controls/malcolmjs
# Feature/socket reconnect #89

## Description

Created a simple wrapper for a native WebSocket which attempts to reconnect at set intervals when connection is lost, whilst also setting all writable attributes to disabled and  setting an alarm state for every attribute. 

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run up an instance of the PandA simulator and pymalcolm server.
- Run npm start and navigate to http://localhost:3000/gui/PANDA//PANDA:SEQ1.
- Close the pymalcolm server and observe disconnect behaviour.
- Restart the pymalcolm server and observe reconnection behaviour.

## Agile board tracking

connect to #89
